src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -455,7 +455,8 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 			if (isEditMode) {
 				this._editor.getControl().focus();
 			} else {
-				(document.activeElement as HTMLElement).blur();
+				const currentCellSelector = `.notebook-cell[id="${this._activeCellId}"] .inputarea.monaco-mouse-cursor-text`;
```

Comment:
So is this a bug in the product as well? If so please update the title/description to specify that.

Could you explain more about the issue here though and how this is fixing it? Why is the activeElement not the input area already? If the mouse cursor is there I would expect that to be the cas.e